cfgov/paying_for_college/models/search.py

```diff
@@ -0,0 +1,32 @@
+from paying_for_college.documents import SchoolDocument
+from search.elasticsearch_helpers import make_safe
+
+
+class SchoolSearch:
+    def __init__(self, search_term):
+        self.search_term = make_safe(search_term).strip()
+        self.results = []
+
+    def autocomplete(self):
+        if self.search_term != '':
+            search = SchoolDocument.search().query(
+                'match', autocomplete=self.search_term)
+            total_results = search.count()
+            self.results = [
+                {'question': str(result.autocomplete), 'url': result.url}
+                for result in search[:total_results]
+            ]
+        return self.results
```

Comment:
The autocomplete method is missing an `execute` step. We also don't need to return all results during the autocomplete result sequence. 10 max, I'd say.

```suggestion
            response = search.execute()
            self.results = [{
                'schoolname': result.text,
                'id': result.school_id,
                'city': result.city,
                'state': result.state}
                for result in response[:10]
            ]
        return self.results
```

Reply:
I fixed the results definition and added the missing `execute` step. I also checked the `autocomplete` method in `search.py` for `ask_cfpb` and that is also missing the `execute` step.